doc-content/enterprise-only/installation/jws-kie-server-verify-proc.adoc

```diff
@@ -0,0 +1,35 @@
+[id='jws-kie-server-verify-proc']
+
+= Verifying the {KIE_SERVER} installation
+
+Verify that {KIE_SERVER} is installed correctly.
+
+.Prerequisites
+* {KIE_SERVER} is installed and configured.
+
+
+. To start {JWS}, enter one of the following commands in the `_JWS_HOME_/tomcat/bin` directory:
++
+** On Linux or UNIX-based systems:
++
+[source,bash]
+----
+$ ./startup.sh
+----
+** On Windows:
++
+[source,bash]
+----
+startup.bat
+----
+. After a few minutes, review the the `_JWS_HOME_/tomcat/logs` directory and correct any errors.
```

Comment:
"review the log files in the...", I think? Reviewing a directory sounds like "ls -l" to me.

Reply:
Done.

Reply:
Done